en/components/general/cli/getting-started-with-angular-schematics.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Step-by-Step Guide | Ignite UI for Angular | Infragistics
+_description: The Step-by-Step mode contains guided experience through the Ignite UI CLI options.
+_keywords: angular cli, ignite ui for angular, infragistics
+---
+
+## Getting Started
```

Comment:
I think the title should be "Getting Started with Ignite UI for Angular Schematics" as it is too general right now and does not correspond to the link and url that lead to the topic.